app/views/layouts/application.html.erb

```diff
@@ -40,7 +40,16 @@
     <%= govuk_header(homepage_url: "/", service_name: t("service.name")) %>
 
     <div class="govuk-width-container">
-      <%= govuk_phase_banner(tag: { text: "Beta" }, text: "This is a new service – your feedback will help us to improve it. ") %>
+      <div class="govuk-phase-banner">
```

Comment:
You can still use the `govuk_phase_banner` to do this - https://govuk-components.netlify.app/components/phase-banner/

Reply:
I am getting a syntax error with this line of code:

`<%= govuk_phase_banner(tag: { text: "Beta" }, text: "This is a new service. Help us improve it and <a class="govuk-link" href="/feedback">give your feedback (opens in new tab)</a>.") %>`